docs/_documentations/mdt-vsc-usingadifferenttemplate.md

```diff
@@ -1,24 +1,20 @@
 ---
 layout: docs
-title: Using a template repository of your own
-description: Using a template repository of your own
-keywords: getting started, setting up, projects, update, help, Theia, test, edit, Theia editor, using own IDE, empty page, refresh, credentials, default editor, Node.js profiling support, code highlighting, JavaScript file, template repository
+title: Configuring template sources
+description: Configuring template sources
+keywords: getting started, setting up, projects, update, help, Theia, test, edit, Theia editor, using own IDE, empty page, refresh, credentials, default editor, Node.js profiling support, code highlighting, JavaScript file, template source
 duration: 1 minute
 permalink: mdt-vsc-usingadifferenttemplate
 type: document
 order: 8
 parent: usingcodewind
 ---
 
-# Using a template repository of your own
+# Configuring template sources
 
-When you create a new project, you can choose from the default set of templates available, or you can choose from a template repository of your own. To add a new repository, edit the `.config/repository_list.json` file within your `codewind-workspace`. The file is a json file that contains an array of objects with two fields, the URL of the repository and a description:
+When you create a new project, you can choose from the default set of templates available, or you can choose from a template source of your own. Use Codewind to manage templates. Templates can make application development easier by providing a structure and boilerplate code to help get a new project started.
 
-```
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Template Source Manager, right-click **Projects (Local)**.
+- Click **Manage Template Sources** to add a source to the table.
```

Comment:
sorry, i missed this one last time.

> - Click **Manage Template Sources** to add a source to the table.

It should be something like.
Click Manage Template Sources to open the Template Sources wizard.
Click Add New to add a source to the table.

Reply:
Added new commit!  👍

Reply:
Added a new commit! 👍